pkg/utils/secrets/basic_auth_test.go

```diff
@@ -81,7 +81,8 @@ var _ = Describe("Basic Auth Secrets", func() {
 				Expect(err).NotTo(HaveOccurred())
 				compareCurrentAndExpectedBasicAuth(obj, expectedBasicAuthObject, false, false)
 			})
-			It("should properly generate Basic Auth Object with Bcrypt hashed password if specified in config", func() {
+			// pending because test is consistently slow, see https://github.com/gardener/gardener/issues/3362
+			PIt("should properly generate Basic Auth Object with Bcrypt hashed password if specified in config", func() {
```

Comment:
Now that #3365 was opened you can revert this change

Reply:
Sure, done.